pkg/maintainer/relaychain.go

```diff
@@ -0,0 +1,13 @@
+package maintainer
+
+import (
+	"github.com/keep-network/keep-core/pkg/bitcoin"
+)
+
+// RelayChain is an interface that provides the ability to communicate with the
+// relay on-chain contract.
+type RelayChain interface {
```

Comment:
What's the point of putting `maintainer.RelayChain` into a separate file? This won't be a huge interface so I'd keep it at the point of use (`relay.go`) to avoid jumping between files. Also, this is a Go-like way of organizing things, we don't need to create a separate file for each public type.

Reply:
Moved it back to `relay.go` in 7e975d796e0dd9de239fc80b0489538061d221ec.
I think the interface will have about 6 methods (there will be more methods than just `Retarget()`, e.g.  `ProofLength()`, `CurrentEpoch()`, `Ready()`, etc. ).